pkg/annotation/annotation.go

```diff
@@ -0,0 +1,13 @@
+// Package annotation contains common Kubernetes metadata. These are defined in
+// https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md.
+package annotation
+
+const (
+	// CordonReasonAnnotationName is the name of the annotation that indicates
+	// the reason of why chart-operator should not apply any update on this chart CR.
+	CordonReasonAnnotationName = "chart-operator.giantswarm.io/cordon-reason"
+
+	// CordonUntilAnnotationName is the name of the annotation that indicates
+	// the expiration date of rule of this cordon.
+	CordonUntilAnnotationName = "chart-operator.giantswarm.io/cordon-until"
```

Comment:
I suggested other names. These stutter now from the callers perspective.

Reply:
@xh3b4sd `CordonExpirationDateAnnotationName` make sense?

Reply:
No, because it would read `annotation.CordonExpirationDateAnnotationName` from the callers perspective. What I would like to see when I import and use the package is `annotation.CordonExpirationDate`. Does that make sense to you?

Reply:
Make sense. 👍